.pre-commit-config.yaml

```diff
@@ -25,16 +25,23 @@ repos:
                 src/process/old/.*\.F90|
                 src/process/Process_driver_template.F90|
                 drivers/ccpp/.*\.F90|
-                parameters/.*\.F90
+                parameters/.*\.F90|
+                src/external/GOCART/.*
```

Comment:
Was this necessary? I would think the submodule should be excluded from the file search.

Reply:
not sure but I thought it better to explicitly do this